content/tutorials/single-page-tutorials/create-nft-with-wallet-lib.md

```diff
@@ -0,0 +1,197 @@
+# Create a NFT with wallet library and IPFS
+
+In this tutorial you will upload your desired image to IPFS and create a NFT from the unique IPFS URL on the Shimmer testnet. This tutorial will build on the tutorial "Setup testnet address and receive funds from faucet" ([IOTA Wiki PR under review](https://github.com/iota-community/iota-wiki/pull/711)). If you haven't completed it, please do so and afterwards come back here. You will use the same folder and development environment as in the mentioned prerequired tutorial, which will allow you to reuse the Stronghold account with funds on it.
+
+## Prerequisites
+
+- Installed [Node.js](https://nodejs.org/en/)
+- Finished [Setup testnet address and receive funds from faucet](https://wiki.iota.org/wallet-setup) tutorial
+
+## Setup
+
+### Prepare development environment
+
+Navigate to the folder of the prerequired tutorial [Setup testnet address and receive funds from faucet](https://wiki.iota.org/wallet-setup):

```

Comment:
```suggestion
Navigate to the folder you created when completing the [set up a testnet address and receive funds from the faucet tutorial](https://wiki.iota.org/wallet-setup):
```

Is it at all possible that someone who already has a running wallet with testnet address would want to start in this tutorial?

Reply:
Yeah. We should mention that option I guess. You are right 🤔